openfl-tutorials/interactive_api/PyTorch_MVTec_PatchSVDD/envoy/mvtec_shard_descriptor.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (C) 2020-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""MVTec shard descriptor."""
+
+import os
+from glob import glob
+from pathlib import Path
+
+import numpy as np
+from imageio import imread
```

Comment:
This dependency is missing in sd_requirements.txt

Reply:
done